### PR TITLE
DI-5531 Readme translation to US English

### PR DESCRIPTION
**Pull request by maheshmuralip**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,6 @@ Any improvements, bug fixes or amendments should also be submitted back as pull
 
 ## Credits
 This plugin was written by Tony Blacker, NHS Leadership Academy Digital Delivery Team. It is based on the tutorial 
-provided by [Morten Rand-Hendriksen](https://mor10.com/) and extended out to match the [NHSUK Frontend library 
-components](https://nhsuk.github.io/nhsuk-frontend/components) 
+provided by [Morten Rand-Hendriksen](https://mor10.com/) and extended out to match the [NHSUK Front-end library 
+components](https://nhsuk.github.io/nhsuk-front-end/components)
```
Comment (tblacker7) [inline README.md]: this will break the link. frontend in this context is a url so please remove hyphenation

File: readme.txt [hunk 2]
```diff
@@ -99,7 +99,7 @@ This plugin has been built specifically for use in the NHS, but it is open sourc
  * added tag component https://service-manual.nhs.uk/design-system/components/tag
  * corrected IE10/11 javascript bug affecting hero placement
  * multiple accessibility improvements, particularly with screen reader and contrast issues
- * updated to NHSUK frontend library 4.1.0 https://github.com/nhsuk/nhsuk-frontend
+ * updated to NHSUK front-end library 4.1.0 https://github.com/nhsuk/nhsuk-front-end
```
Comment (tblacker7) [inline readme.txt]: again, this is a url, please remove hyphenation of frontend

**Review by tblacker7 — CHANGES_REQUESTED** [review]
couple of url's need hyphenation of front-end removing otherwise they will become broken links. Otherwise good.

File: README.md (hunk 1 above)
Comment (maheshmuralip) [reply README.md]: Done, thanks.

File: readme.txt (hunk 2 above)
Comment (maheshmuralip) [reply readme.txt]: Done, thanks.

**Review by tblacker7 — APPROVED** [review]

**Pull request merged**